cmd/init.go

```diff
@@ -78,7 +78,7 @@ func Init() *cobra.Command {
 	cmd.Flags().StringVarP(&opts.Context, "context", "c", "", "context target for generating the okteto manifest")
 	cmd.Flags().StringVarP(&opts.DevPath, "file", "f", utils.DefaultManifest, "path to the manifest file")
 	cmd.Flags().BoolVarP(&opts.Overwrite, "replace", "r", false, "overwrite existing manifest file")
-	cmd.Flags().BoolVarP(&opts.Version1, "v1", "", false, "create a v1 okteto manifest: www.okteto.com/docs/reference/manifest-v1/")
+	cmd.Flags().BoolVarP(&opts.Version1, "v1", "", false, "create a v1 okteto manifest: www.okteto.com/docs/0.10/reference/manifest/")
```

Comment:
The 0.10 version planned to be be removed in 3 months. If we want to have a reference to the v1 manifest, can we put that page somewhere else? Otherwise this link will be broken pretty soon. 

We could put it somewhere in the "current" version of the docs. We can create a "v1" subsection in the manifest reference section, if this is something that people need. Another option is to put it in this GitHub repo.